### PR TITLE
Swapping benchmarks to use VMVX instead of VMLA.

### PR DESCRIPTION
**Pull request by benvanik**
VMVX MobileBERT is disabled for now until it can be sped up.

### CODE REVIEW

**Comment by benvanik** [comment]
@hanhanW I don't know how the benchmark infra works - is this correct?

File: build_tools/mako/configuration.py [hunk 1]
```diff
@@ -127,6 +123,16 @@ def get_pixel4_default_target_list(skipped_target=None,
                  runtime_flags=[
                      "--task_topology_group_count=3",
                  ]),
+      TargetInfo(driver="vmvx",
+                 hal_target_backend="vmvx",
+                 taskset="f0",
+                 mako_tag="vmvx3t",
```
Comment (hanhanW) [inline build_tools/mako/configuration.py:129]: Do you intend to add a benchmark on vmvx with 3 threads? This should match to the `value_key` in mako config files.

**Review by hanhanW — CHANGES_REQUESTED** [review]
> @hanhanW I don't know how the benchmark infra works - is this correct?

The change in config files looks good, but it will need me to run mako update commands. Let me land the PR and I will make sure they are updated before merging?

File: build_tools/mako/configuration.py (hunk 1 above)
Comment (benvanik) [reply build_tools/mako/configuration.py:129]: yeah we'll want it threaded - will do

**Comment by benvanik** [comment]
> > @hanhanW I don't know how the benchmark infra works - is this correct?
> 
> The change in config files looks good, but it will need me to run mako update commands. Let me land the PR and I will make sure they are updated before merging?

so you want to take this over this PR? (that works for me! :)

**Comment by hanhanW** [comment]
> > > @hanhanW I don't know how the benchmark infra works - is this correct?
> > 
> > 
> > The change in config files looks good, but it will need me to run mako update commands. Let me land the PR and I will make sure they are updated before merging?
> 
> so you want to take this over this PR? (that works for me! :)

Yes, I can take it over. I can prepare the PR based on the configuration you have. One question, do we want to track both singled threaded and 3-threaded for vmvx backend?

**Comment by benvanik** [comment]
> > > > @hanhanW I don't know how the benchmark infra works - is this correct?
> > > 
> > > 
> > > The change in config files looks good, but it will need me to run mako update commands. Let me land the PR and I will make sure they are updated before merging?
> > 
> > 
> > so you want to take this over this PR? (that works for me! :)
> 
> Yes, I can take it over. I can prepare the PR based on the configuration you have. One question, do we want to track both singled threaded and 3-threaded for vmvx backend?

cool - updated with the right key.

I'm not too interested in single threaded - it's slow enough without handicapping it :)
(it should eventually be faster than vmla, but it requires work #5880)

File: build_tools/mako/configuration.py [hunk 2]
```diff
@@ -270,7 +282,7 @@ def get_s20_default_target_list(skipped_target=None,
                 name="S20",
                 benchmark_key="4636549841944576",
                 targets=get_s20_default_target_list(
-                    skipped_target=['cpu', 'vmla', 'cpu2', 'vlk2'])),
+                    skipped_target=['cpu', 'vmvx', 'cpu2', 'vlk2'])),
```
Comment (hanhanW) [inline build_tools/mako/configuration.py:285]: This should be `vmvx3t`. (the variable name is confusing, the target means benchmark target, ie, mako_tag, in this case...).

**Review by hanhanW — APPROVED** [review]
Just one nit, then I can help land it.

note: it's not swapping benchmarks to VMVX. This will stop tracking VMLA, and add a new metric/plot for VMVX. The track for VMVX won't inherit the performance data from VMLA.

File: build_tools/mako/configuration.py (hunk 2 above)
Comment (benvanik) [reply build_tools/mako/configuration.py:285]: ah - gotcha

**Comment by benvanik** [comment]
> Just one nit, then I can help land it.
> 
> note: it's not swapping benchmarks to VMVX. This will stop tracking VMLA, and add a new metric/plot for VMVX. The track for VMVX won't inherit the performance data from VMLA.

👍  they are totally different beasts so I think cutting over is good

**Comment by hanhanW** [comment]
Triggered a benchmark run: https://buildkite.com/iree/iree-android-benchmark/builds/2568

File: build_tools/mako/configuration.py [hunk 3]
```diff
@@ -270,7 +282,7 @@ def get_s20_default_target_list(skipped_target=None,
                 name="S20",
                 benchmark_key="4636549841944576",
                 targets=get_s20_default_target_list(
-                    skipped_target=['cpu', 'vmla', 'cpu2', 'vlk2'])),
```
Comment (hanhanW) [inline build_tools/mako/configuration.py:273]: We should also skip vmvx3t in l.233 and l.237

**Review by hanhanW — CHANGES_REQUESTED** [review]

**Comment by benvanik** [comment]
updated
(getting close to landing vmla removal - the benchmarks will break without it, so would be good to land this)

**Comment by hanhanW** [comment]
Triggered another benchmark run: https://buildkite.com/iree/iree-android-benchmark/builds/2570

Once it's passed, I'll run update commands and land the PR.

**Comment by hanhanW** [comment]
FYI, it takes 19.5 sec to finish MobileNetV2 on Pixel 4. I'm going to run update commands and land the PR.

**Review by hanhanW — APPROVED** [review]

**Comment by benvanik** [comment]
Thanks @hanhanW!

**Pull request merged**